templates/bundles/TwigBundle/Exception/error.html.twig

```diff
@@ -1,61 +1,55 @@
 {% extends 'base.html.twig' %}
-
 {% block stylesheets %}
-    {{ encore_entry_link_tags('exception') }}
+    {{ encore_entry_link_tags('web-site') }}
+    <style>
```

Comment:
please try to avoid this

Reply:
What it's wrong here? Using style tag or what? Coding request it's not clear :P

Reply:
avoid "<style>"  I guess you can create those pages using only [bulma](https://bulma.io/) without any extra css